pandas/core/resample.py

```diff
@@ -205,9 +205,7 @@ def __getattr__(self, attr: str):
 
     # error: Signature of "obj" incompatible with supertype "BaseGroupBy"
     @property
-    def obj(self) -> NDFrameT:  # type: ignore[override]
-        # error: Incompatible return value type (got "Optional[Any]",
-        # expected "NDFrameT")
```

Comment:
Can you keep the comment containing the mypy error?

Reply:
Alright